adapters/gst/gst_plugins/python/zeromq_sink.py

```diff
@@ -177,6 +178,16 @@ def do_render(self, buffer: Gst.Buffer):
 
     def do_stop(self):
         """Stop source."""
+
+        if self.socket_type == SenderSocketTypes.DEALER:
+            self.logger.info('Sending End-of-Stream message to socket %s', self.socket)
+            self.sender.send_multipart([END_OF_STREAM_MESSAGE])
+            self.logger.info(
+                'Waiting for End-of-Stream message confirmation from socket %s',
+                self.socket,
+            )
+            self.sender.recv()
```

Comment:
@tomskikh I would propose setting a reasonable timeout here (like 10 seconds). And to report with ok (0) zero code or err (1) and message if timeout happened.

Reply:
@bwsw Added a timeout.